### PR TITLE
[@types/luxon] new add new types for weekdayShort & weekdayLong 

### PR DESCRIPTION
**Pull request by alix1383**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [ ] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/luxon/src/datetime.d.ts [hunk 1]
```diff
@@ -196,6 +196,24 @@ export type WeekNumbers =
...
+type weekdayShort =
+    |'Mon'
+    |'Tue'
+    |'Wed'
+    |'Thu'
+    |'Fri'
+    |'Sat'
+    |'Sun';
+
+type weekdayLong =
```
Comment (codershiba) [inline types/luxon/src/datetime.d.ts:208]:
```suggestion
export type weekdayLong =
```

File: types/luxon/src/datetime.d.ts [hunk 2]
```diff
@@ -196,6 +196,24 @@ export type WeekNumbers =
     | 52
     | 53;
 
+type weekdayShort =
```
Comment (codershiba) [inline types/luxon/src/datetime.d.ts:199]:
```suggestion
export type weekdayShort =
```

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/luxon/src/datetime.d.ts

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against a919bcb48f3e8b345cff165e28e687676541a8fa
</p>

**Review by ghost — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Comment by CarsonF** [comment]
These are the strings for English, but not for other locales.
```ts
DateTime.now().setLocale('fr').weekdayLong // => 'dimanche'
```
If you are looking for these English literal types, you'll have to do something like this in your own codebase.

**Comment by alix1383** [comment]
> These are the strings for English, but not for other locales.
> 
> ```ts
> DateTime.now().setLocale('fr').weekdayLong // => 'dimanche'
> ```
> 
> If you are looking for these English literal types, you'll have to do something like this in your own codebase.

Thank you for reminding me that I had forgotten about this.

**Pull request closed without merging**